### ISSUE TITLE
Dash license check should also run on "push to main"

### ISSUE THREAD

**Issue by SebastianSchildt** [eclipse-kuksa/kuksa-android-companion#32]
https://github.com/eclipse-kuksa/kuksa-android-companion/blob/main/.github/workflows/dash.yaml

Dash license check runs only on PRs, it should also run on push to main.

Usecase: It runs in PR, maybe some stuff not cleared, you may choose to clear first, but once it is merged into main it would be good it runs again, so you can find the dash data for recent main in GH Actions easily
